### PR TITLE
Made the server exit after first worker crash.

### PR DESCRIPTION
**Pull request by psenchanka**

### CODE REVIEW

**Comment by accek** [comment]
What about using atexit to cleanly close the DB in case of SIGTERM?

**Comment by psenchanka** [comment]
According to https://docs.python.org/3.6/library/atexit.html#module-atexit, `atexit` isn't executed for signals not handled by Python, and IIRC SIGTERM is one of them.

I've added a simple signal header for workers.

**Comment by psenchanka** [comment]
After some manual testing, it seems that this `worker_exit` hook covers all scenarios of worker exiting, including timeouts (SIGABRT), SIGINT, SIGQUIT, and SIGTERM.

File: filetracker/servers/run.py [hunk 1]
```diff
@@ -85,12 +86,12 @@ def main(args=None):
...
-        |   print('Exiting gunicorn: worker received SIGINT')
-        |   sys.exit(1)
+        |    print('Exiting gunicorn: worker received SIGINT')
+        |    os.kill(os.getppid(), signal.SIGTERM)
         |
         |def worker_abort(worker):
-        |   print('Exiting gunicorn: worker received SIGABRT (timeout?)')
-        |   sys.exit(1)
+        |    print('Exiting gunicorn: worker received SIGABRT (timeout?)')
+        |    os.kill(os.getppid(), signal.SIGTERM)
```
Comment (accek) [inline filetracker/servers/run.py]: What about a comment on why this is needed?

**Pull request merged**